src/test/java/org/terasology/launcher/game/TestGameStarter.java

```diff
@@ -50,7 +58,20 @@ public void testConstruction() {
     }
 
     private GameStarter newStarter() {
-        return new GameStarter(gamePath, gameDataPath, HEAP_MIN, HEAP_MAX, javaParams, gameParams, LOG_LEVEL);
+        GameRelease release;
+        try {
+            release = new GameRelease(
+                    new GameIdentifier("5.1.0", Build.STABLE, Profile.OMEGA),
+                    new URL("https://repository.example"),
+                    new ReleaseMetadata(
+                            "# CHANGES",
+                            (new Calendar.Builder()).setDate(2021, 1, 1).build().getTime(),
+                            true)
+            );
+        } catch (MalformedURLException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
😭 eight lines of imports and twelve lines

Reply:
:shrug: as said in another comment: suggestions are welcome :upside_down_face: